lib/entry.js

```diff
@@ -235,6 +258,24 @@ function SalesforceEntity(callScope) {
     });
   };
 
+  this.listLinkedObjects = function listLinkedObjects(conf, next) {
+    // eslint-disable-next-line consistent-return
+    self.refreshToken(_.clone(conf), async (err, newCfg) => {
+      if (err) {
+        return next(err);
+      }
+      try {
+        newCfg.sobject = newCfg.object;
+        const metaLoader = new MetaLoader(newCfg, this);
+        const relatedObjs = await metaLoader.getLinkedObjectsModel();
+        console.log(relatedObjs);
```

Comment:
please, use logger instead of console.log

Reply:
Oops! I forgot to delete that statement